README.md

```diff
@@ -37,17 +58,11 @@ Javadocs and Scaladocs are hosted on GitHub:
 
 ## Building
 
-### Build Dependencies
+The faunadb-jvm project is built using sbt:
 
 * **sbt**: [Scala Simple Build Tool](http://www.scala-sbt.org/)
 
-### Building and Using
-
-1. Clone this repository.
-2. Run `sbt package` to build all JAR files needed.
-3. Copy the `faunadb-httpclient/target/faunadb-httpclient-0.1-SNAPSHOT.jar` to your project's unmanaged library directory.
-4. Copy the client JAR of your choice to your project's unmanaged library directory. `faunadb-java/target/faunadb-java-0.1-SNAPSHOT.jar`
-for the Java client, and `faunadb-scala/target/scala-2.11/faunadb-scala_2.11-0.1-SNAPSHOT.jar` for the Scala client.
+To build and run tests, run `sbt test` from the project root.
```

Comment:
This should discuss how to set up their env vars in order to get access to either cloud or enterprise.